modules/snooker/src/main/ts/ephox/snooker/resize/Recalculations.ts

```diff
@@ -30,7 +30,7 @@ interface CellHeightSpan extends CellHeight {
 
 // Returns an array of all cells in warehouse with updated cell-widths, using
 // the array 'widths' of the representative widths of each column of the table 'warehouse'
-const recalculateWidth = function (warehouse: Warehouse, widths: number[]): CellWidthSpan[] {
+const recalculateWidthForTd = (warehouse: Warehouse, widths: number[]): CellWidthSpan[] => {
```

Comment:
Instead of `td` maybe use `Cells` here? As this could be a `th` as well, so the name is misleading.

Reply:
Seems sensible.